bokehjs/src/coffee/api/figure.coffee

```diff
@@ -41,14 +40,14 @@ define [
         try
           new Date.parse(range.get('start'))
           return Collections("DatetimeAxis")
-        catch
+        catch e
```

Comment:
Not necessarily better, but it was getting in my way. Now I see that I'm just using outdated coffee script compiler.